src/Lecture1.hs

```diff
@@ -119,4 +153,14 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list1 = (show n) ++ " is greater than " ++ (show (fst nums)) ++ " elements and lower than " ++ (show (snd nums)) ++ " elements"
+    where
+        lg :: [Int] -> Int -> (Int, Int) -> (Int, Int)
+        lg list bound accu
+            | null list = accu
+            | (head list < bound) = lg (tail list) bound ((fst accu)+1, (snd accu))
+            | (head list > bound) = lg (tail list) bound ((fst accu), (snd accu)+1)
+            | otherwise = lg (tail list) bound ((fst accu), (snd accu))
+        nums = lg list1 n (0, 0)
```

Comment:
I see that the number `n` that you pass in recursive calls doesn't change. You always pass the same number. In that case, you can remove this extra argument and use `n` from the function instead. It's possible because functions in `where` can see all the arguments of the top-level function 🔍